lib/identity/account.rb

```diff
@@ -176,17 +176,17 @@ class Account < Sinatra::Base
         begin
           api = HerokuAPI.new(ip: request.ip, request_ids: request_ids,
             version: 2)
-          res = api.post(path: "/auth/finish_reset_password/#{token}",
-            expects: 200, body: URI.encode_www_form({
-              :password              => params[:password],
-              :password_confirmation => params[:password_confirmation],
-            }))
+          api.post(path: "/auth/finish_reset_password/#{token}",
+                   expects: 200, body: URI.encode_www_form({
```

Comment:
I move the `URI.encode_www_form` line out to a variable and that should look better now (`--amend`)